modules/javascript/src/main/webapp/javascript/atmosphere.js

```diff
@@ -3104,12 +3104,17 @@
 
             // encodeURI and decodeURI are needed to normalize URL between IE and non-IE,
             // since IE doesn't encode the href property value and return it - http://jsfiddle.net/Yq9M8/1/
-            var ua = window.navigator.userAgent;
+
+          var ua = window.navigator.userAgent;
             if(ua.indexOf('MSIE ')>0 || ua.indexOf('Trident/') > 0 || ua.indexOf('Edge/') > 0){
-                return encodeURI(decodeURI(div.firstChild.href));
+                return atmosphere.util.fixedEncodeURI(decodeURI(div.firstChild.href));
             }
             return div.firstChild.href;
         },
+		
+        fixedEncodeURI: function (str) {
+			return encodeURI(str).replace(/%5B/g, '[').replace(/%5D/g, ']');
+		},
```

Comment:
Minor, but can you make sure tabs are properly set (format)

Reply:
Still not workking (the format).

Reply:
Yes, I know. sorry for these problems.  I had some windows problems. On my computer seems to be ok, but here in github  no...:(. 

Now its ok. 

Thank you so much for your patient